packages/theme/src/theme/_forms.scss

```diff
@@ -338,6 +365,20 @@ form {
 			line-height: 1.5;
 		}
 	}
+
+	.form-group > input.form-control::placeholder {
+		color: $silver;
+	}
+
+	.form-control + label {
+		font-size: 14px;
```

Comment:
A var exists in bootstrap-theme with this value.

Reply:
fixed